core/options.hpp

```diff
@@ -133,6 +133,17 @@ struct Options
 			 "Camera mode as W:H:bit-depth:packing, where packing is P (packed) or U (unpacked)")
 			("viewfinder-mode", value<std::string>(&viewfinder_mode_string),
 			 "Camera mode for preview as W:H:bit-depth:packing, where packing is P (packed) or U (unpacked)")
+			("buffer-count", value<unsigned int>(&buffer_count)->default_value(0), "override bufferCount config (0 use imlp default)")
+			("autofocus-mode", value<std::string>(&afMode)->default_value("manual"),
+			 " Control to set the mode of the AF (autofocus) algorithm.(manual, auto, continuous)")
+			("autofocus-range", value<std::string>(&afRange)->default_value("normal"),
+			 "Set the range of focus distances that is scanned.(normal, macro, full)")
+			("autofocus-speed", value<std::string>(&afSpeed)->default_value("normal"),
+			 "Control that determines whether the AF algorithm is to move the lens as quickly as possible or more steadily.(normal, fast)")
+			("autofocus-window", value<std::string>(&afWindow)->default_value("0,0,0,0"), 
+			"Sets AfMetering to  AfMeteringWindows an set region used for examle  e.g. 0.25,0.25,0.5,0.5")
+			("lens-position", value<float>(&lens_position)->default_value(0),
+			 "Acts as a control to instruct the lens to move to a particular position and also reports back the position of the lens for each frame. (0 moves the lens to infinity;0.5 moves the lens to twice the hyperfocal distance;1 moves the lens to the hyperfocal position )")
```

Comment:
We are in discussion about the exact units for the lens-position control.  Right now it is based on hyperfocal distance, but we probably will recommend changing to diopters.  I suggest you remove this for now to avoid any confusion.

Reply:
For now I would remove the option until the actual definition is finalised by libcamera.

Reply:
To clarify, this is an option that should be in once the definition has been finalised by libcamera.